### PR TITLE
Avoid using unistd.h and sleep

### PR DESCRIPTION
**Pull request by masterleinad**
This PR aims to remove any usage of the platform-dependent `sleep`.

### CODE REVIEW

File: tests/mpi/simple_mpi_01.cc [hunk 1]
```diff
@@ -39,7 +39,6 @@ test_mpi()
   for (unsigned int i = 1; i < numprocs; ++i)
     {
       MPI_Barrier(MPI_COMM_WORLD);
-      //      system("sleep 1");
```
Comment (drwells) [inline tests/mpi/simple_mpi_01.cc]: :)

**Comment by drwells** [comment]
Nice catch!

**Comment by bangerth** [comment]
Ah, great. Definitely ok to merge from my side!

**Pull request merged**